package.json

```diff
@@ -84,10 +83,14 @@
 	},
 	"jest": {
 		"preset": "ts-jest",
+		"globals": {
+			"ts-jest": {
+				"diagnostics": false
```

Comment:
Quick question: what does this do?

Reply:
It prevents ts-jest from checking our typescript for compiler errors. We have a separate script for that, we just need ts-jest to run the tests.